doc/ale.txt

```diff
@@ -286,6 +288,45 @@ Default: `'-Wall'`
 
 This variable can be changed to modify flags given to gcc.
 
+-------------------------------------------------------------------------------
+4.5. fortran-gcc                               *ale-linter-options-fortran-gcc*
+
+g:ale_fortran_gcc_options                           *g:ale_fortran_gcc_options*
+
+Type: |String|
+Default: `'-Wall'`
+
+This variable can be changed to modify flags given to gcc.
+
+-------------------------------------------------------------------------------
+4.6. html-tidy                                   *ale-linter-options-html-tidy*
+
+g:ale_html_tidy_executable                         *g:ale_html_tidy_executable*
+
+Type: |String|
+Default: `'tidy'`
+
+This variable can be changed to change the path to tidy.
+
+
+g:ale_html_tidy_args                                     *g:ale_html_tidy_args*
+
+Type: |String|
+Default: '-q -e -language en'
+
+This variable can be changed to change the arguments provide to the
+executable.
+
+
+File encoding
+The linter will auto-detect the file encoding and provide the good option to
+the executable. If it can't detect the file encoding, it will fall back to
+utf8.
```

Comment:
I'd remove line 321 here with the words "File encoding," as there isn't any sub-subsection markup. The following sentences don't make the best use of English, but the ideas are there. I have an alternative suggestion.

"ALE will attempt to automatically detect the appropriate file encoding to provide to html-tidy, and fall back to UTF-8 when encoding detection fails."